paradise_papers_search/fetch_api/models/Entity.py

```diff
@@ -19,7 +19,61 @@ class Entity(DjangoNode):
     status                   = StringProperty()
     officers                 = RelationshipFrom('.Officer.Officer', 'OFFICER_OF')
     intermediaries           = RelationshipFrom('.Intermediary.Intermediary', 'INTERMEDIARY_OF')
-    addressess               = RelationshipTo('.Address.Address', 'REGISTERED_ADDRESS')
+    addresses                = RelationshipTo('.Address.Address', 'REGISTERED_ADDRESS')
     others                   = RelationshipFrom('.Other.Other', 'CONNECTED_TO')
 
+    @property
+    def serialize(self):
+        return {
+            'properties': {
+                'sourceID': self.sourceID,
+                'address': self.address,
+                'jurisdiction': self.jurisdiction,
+                'service_provider': self.service_provider,
+                'countries': self.countries,
+                'jurisdiction_description': self.jurisdiction_description,
+                'valid_until': self.valid_until,
+                'ibcRUC': self.ibcRUC,
+                'name': self.name,
+                'country_codes': self.country_codes,
+                'incorporation_date': self.incorporation_date,
+                'node_id': self.node_id,
+                'status': self.status,
+            },
+        }
+
+    @property
+    def serialize_connections(self):
+        return [
+            {
+                'nodes_type': 'officer',
+                'nodes_related': [officer.serialize for officer in self.officers.all()],
+            },
+            {
+                'nodes_type': 'intermediary',
+                'nodes_related': [intermediary.serialize for intermediary in self.intermediaries.all()],
+            },
+            {
+                'nodes_type': 'address',
+                'nodes_related': [address.serialize for address in self.addresses.all()],
+            },
+            {
+                'nodes_type': 'other',
+                'nodes_related': [other.serialize for other in self.others.all()],
+            },
+            {
+                'nodes_type': 'entity',
+                'nodes_related': [entity.serialize for entity in self._entity_relationships()],
+            },
+        ]
+
+    def _entity_relationships(self):
+        results         = self.cypher("START p=node({self}) MATCH n=(p)<-[r]->(x:Entity) RETURN r, x.node_id as Node_id")
+        relationships   = []
+
+        for row in results[0]:
+            relationships.append(self.nodes.get(node_id=row[1]))
```

Comment:
Good work getting the node through Neomodel, now you can take advantage of the serialize methods.